client/src/components/search-page/Dogs.js

```diff
@@ -1,24 +1,18 @@
 import "./Dogs.css";
-import { getItemsFromLocalStorage } from "../../utils/itemStorage";
-import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 import { GiSniffingDog } from "react-icons/gi";
 import RandomNumber from "../../utils/RandomNumber";
 import FakeFrens from "../../utils/FakeFrens";
+import useFriends from "../../hooks/useFriends";
+import { getDogsFilteredByName } from "../../utils/dogs";
 
 export default function Dogs({ filter }) {
-  const [friends, setFriends] = useState([]);
-  useEffect(() => {
-    const friendsList = getItemsFromLocalStorage("friends");
-    setFriends(friendsList);
-  }, []);
-
-  const filteredName = friends.filter(
-    (friend) => friend.name.toLowerCase().includes(filter) || filter === ""
-  );
+  const [friends, setFriends] = useFriends();
+  console.log(setFriends); // where to use this?
```

Comment:
same comment as above. If you don't want the setState, don't have your custom hook return it